sw/device/silicon_creator/rom_ext/e2e/ownership/BUILD

```diff
@@ -0,0 +1,47 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load(
+    "//rules/opentitan:defs.bzl",
+    "fpga_params",
+    "opentitan_test",
+)
+
+package(default_visibility = ["//visibility:public"])
+
+opentitan_test(
+    name = "ownership_transfer_test",
```

Comment:
Should we start maintaining a test plan for ownership transfer? 

Similar to sw/device/silicon_creator/rom/data/rom_e2e_testplan.hjson. I think it is ok if we add it as a separate test plan. 

It will be useful certification later on.

Reply:
This is the current ownership transfer test plan:
https://github.com/lowRISC/opentitan/blob/earlgrey_es_sival/sw/device/silicon_creator/rom_ext/data/rom_ext_e2e_testplan.hjson#L221-L488 

This covers basic functionality; I think more test cases are needed however.